### PR TITLE
Docker compose and dockerfile updates

### PR DESCRIPTION
**Pull request by orangewolf**
get a building dockerfile and a docker-compose to make building and pushing easy

### CODE REVIEW

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -1,25 +1,26 @@
-# we are running the deploy through vercel now so we don't need this github action
-# TODO(alishaevn): fix any potential sentry error's if we switch from vercel to an in house deploy infrastructure
+we are running the deploy through vercel now so we don't need this github action
+TODO(alishaevn): fix any potential sentry error's if we switch from vercel to an in house deploy infrastructure
```
Comment (alishaevn) [inline .github/workflows/deploy.yml:3]:
```suggestion
```

Comment (alishaevn) [reply .github/workflows/deploy.yml:3]: assuming you meant to delete this entirely.

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -1,78 +1,24 @@
...
-#         with:
-#           cache-from: type=registry,ref=${{ env.REGISTRY }}/${{ env.REPO_LOWER }}:${{ env.TAG }}
-#           context: .
-#           push: true
-#           tags: |
-#             ${{ steps.meta-docker-image.outputs.tags }}
-#             ${{ env.REGISTRY }}/${{ env.REPO_LOWER }}:${{ env.TAG }}
+jobs:
+  build:
+    uses: scientist-softserv/actions/.github/workflows/build.yaml@v0.0.14
```
Comment (alishaevn) [inline .github/workflows/build.yml:20]: we're using `v0.0.15` in the deploy file. is this intentional?

File: .gitignore [hunk 3]
```diff
@@ -7,9 +7,13 @@ node_modules/
 # Where ENV secrets are stored. refer to the "Note" at the link below for details on the use of this file
 # https://nextjs.org/docs/basic-features/environment-variables#default-environment-variables
 .env.local
+.env.development
```
Comment (alishaevn) [inline .gitignore:10]: why'd you add this file? what's in it?

File: ops/production-deploy.tmpl.yaml [hunk 4]
```diff
@@ -0,0 +1,36 @@
+nextAuthUrl: "https://phenovista.softserv.cloud/api/auth"
```
Comment (alishaevn) [inline ops/production-deploy.tmpl.yaml:1]: is this temporary? (along with the other references to it below) they want the site to point at store.phenovista.com.

ref: [this ticket](https://github.com/scientist-softserv/phenovista-digital-storefront/issues/17)

File: .github/workflows/build.yml (hunk 2 above)
Comment (orangewolf) [reply .github/workflows/build.yml:20]: no, it can be 0.0.15. they do the same thing for this part

File: .gitignore (hunk 3 above)
Comment (orangewolf) [reply .gitignore:10]: .env.development is our standard env file for all apps. it means we can use stack_car on it. .env.local is similar in purpose, but just trying to keep things consistant

File: ops/production-deploy.tmpl.yaml (hunk 4 above)
Comment (orangewolf) [reply ops/production-deploy.tmpl.yaml:1]: yes, I dont have access to that url so we need something to get the deploy all figured out

File: .github/workflows/build.yml (hunk 2 above)
Comment (alishaevn) [reply .github/workflows/build.yml:20]:
```suggestion
    uses: scientist-softserv/actions/.github/workflows/build.yaml@v0.0.15
```

**Review by alishaevn — CHANGES_REQUESTED** [review]

File: .github/workflows/build.yml [hunk 5]
```diff
@@ -1,78 +1,24 @@
...
-#         with:
-#           cache-from: type=registry,ref=${{ env.REGISTRY }}/${{ env.REPO_LOWER }}:${{ env.TAG }}
-#           context: .
-#           push: true
-#           tags: |
-#             ${{ steps.meta-docker-image.outputs.tags }}
-#             ${{ env.REGISTRY }}/${{ env.REPO_LOWER }}:${{ env.TAG }}
+jobs:
+  build:
+    uses: scientist-softserv/actions/.github/workflows/build.yaml@v0.0.15
```
Comment (alishaevn) [inline .github/workflows/build.yml:20]:
```suggestion
    uses: scientist-softserv/actions/.github/workflows/build.yaml@v0.0.14
```

**Review by alishaevn — APPROVED** [review]

**Comment by alishaevn** [comment]
ref:
- https://github.com/scientist-softserv/webstore/issues/322

**Pull request merged**